src/reducers/notifications/index.ts

```diff
@@ -1,27 +1,56 @@
 import { AnyAction } from 'redux'
 import { reject } from 'ramda'
-import { Notification } from './types'
+import { Notification } from 'src/lib/notifications'
 
 export const SCHEDULE_NOTIFICATION = 'SCHEDULE_NOTIFICATION'
 export const REMOVE_NOTIFICATION = 'REMOVE_NOTIFICATION'
+export const SET_ACTIVE_NOTIFICATION = 'SET_ACTIVE_NOTIFICATION'
 export const CLEAR_NOTIFICATIONS = 'CLEAR_NOTIFICATIONS'
 
-export type NotificationsState = Notification[]
+export type NotificationsState = {
+  queue: Notification[]
+  active?: Notification | null
```

Comment:
Does the key need to be both optional, and potentially `null`?

Reply:
no, will make it not optional and null by default :+1: